aries-site/src/layouts/content/Example/ExampleControls.js

```diff
@@ -78,6 +86,19 @@ export const ExampleControls = ({
             />
           </Box>
         )}
+        {componentCode && (
```

Comment:
Naming ideas:
- sourceCode
- grommetSource
- others?

Otherwise, I know I am always going to need to look up which is which.